pkg/kubelet/stats/BUILD

```diff
@@ -3,13 +3,13 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 go_library(
     name = "go_default_library",
     srcs = [
-        "cadvisor_stats_provider.go",
```

Comment:
why this file name change? Type is still called cadvisorStatsProvider so no need to change the name of the file.